src/localization/i18n.ts

```diff
@@ -0,0 +1,4 @@
+// IMPORTANT: This file adds comments recognized by the react-i18next-parser so that
+// labels declared in console-extensions.json are added to the message catalog.
+
+// t("NodeHealthchecks")
```

Comment:
you need to add i18n keys from `console-extensions.json` here

```suggestion
// t("plugin__node-remediation-console-plugin~NodeHealthChecks")
...and so on
```